CDTDatastore/touchdb/TDReplicator.m

```diff
@@ -507,6 +507,11 @@ - (BOOL)goOffline
     if (!_online) return NO;
     CDTLogInfo(CDTREPLICATION_LOG_CONTEXT, @"%@: Going offline", self);
     _online = NO;
+    NSString *msg = @"Reachability tracker has determined we have gone offline. Stopping replication.";
```

Comment:
I think this message would benefit from being a little less colloquial. Also I think given that it isn't logged immediately but IIUC arrives in the completion handler the tense of "stopping" may be incorrect.
Maybe:
>Replication stopped because the reachability tracker determined the device was offline.

WDYT?